webpack.config.js

```diff
@@ -202,15 +213,15 @@ module.exports = ({ production } = {}, { extractCss, analyze, tests, hmr, port,
   },
   performance: { hints: false },
   devServer: {
-    contentBase: outDir,
+    static: outDir,
     // serve index.html for all 404 (required for push-state)
     historyApiFallback: true,
     open: project.platform.open,
     hot: hmr || project.platform.hmr,
     port: port || project.platform.port,
     host: host
   },
-  devtool: production ? 'nosources-source-map' : 'cheap-module-eval-source-map',
+  devtool: production ? undefined : 'cheap-module-source-map',
```

Comment:
Copy pasted from latest Aurelia-cli project, that uses webpack5

I can dig deeper and research other options